include/swift/SILOptimizer/Utils/Existential.h

```diff
@@ -86,6 +86,11 @@ struct ConcreteExistentialInfo {
 
   ConcreteExistentialInfo(ConcreteExistentialInfo &) = delete;
 
+  // Empty initilization of ConcreteExistentialInfo. This is used when the
+  // concrete type is determined via ProtocolConformanceAnalysis. The fields
+  // are populated during sil_combiner.
+  ConcreteExistentialInfo() {}
```

Comment:
Instead of default initializing, can you pass around an `Optional<ConcreteExistentialInfo>`?